### PR TITLE
hiding wip for deployment

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
For MVP1 Bureau:
- [x] hide new Settings page
- [x] hide non-built Bureau pages
- [x] Hide Bureau Position Details Export button
- [x] Hide  Bureau Position Details Print button
- [x] Post -> Location

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
🦜Bureau MVP1🦜

File: src/Components/BureauPage/PositionManager/PositionManager.jsx [hunk 1]
```diff
@@ -182,9 +182,9 @@ const PositionManager = props => {
                       />
                     </div>
                     <div className="filter-div">
-                      <div className="label">Post:</div>
+                      <div className="label">Location:</div>
                       <Picky
-                        placeholder="Select Post(s)"
+                        placeholder="Select Location(s)"
```
Comment (mjoyce91) [inline src/Components/BureauPage/PositionManager/PositionManager.jsx:187]: 👍

File: src/Constants/Menu.js [hunk 2]
```diff
@@ -95,11 +95,12 @@ export const GET_PROFILE_MENU = () => MenuConfig([
...
-        text: 'Settings',
-        route: '/profile/settings/',
-        icon: 'cogs',
-      },
+      checkFlag('flags.static_content') ?
+        {
+          text: 'Settings',
+          route: '/profile/settings/',
+          icon: 'cogs',
+        } : null,
```
Comment (mjoyce91) [inline src/Constants/Menu.js:103]: 👍

**Review by mjoyce91 — APPROVED** [review]
nice!

**Pull request merged**